### ISSUE TITLE
[FEA] Support escapechar in read_csv

### ISSUE THREAD

**Issue by mjsamoht** [rapidsai/cudf#715]
This corresponds to 'escapechar' parameter in pandas.  One-character string used to escape delimiter when quoting is QUOTE_NONE.

**Comment by OlivierNV** [comment]
It seems like quoting only has the true/false value (unlike Panda's 0/1/2/3), but it seems that supporting escapechar to escape the delimiter when outside a quote would be relatively straightforward (escaping end-of-line would have much bigger ramifications)

**Comment by harrism** [comment]
@mjsamoht is this still an issue?

**Comment by mjsamoht** [comment]
> @mjsamoht is this still an issue?

Yes, this is still open.  We never implemented escape character support in CSV reader.  But I believe this is very low priority since nobody asked for it.
